aries_cloudagent/config/argparse.py

```diff
@@ -1160,14 +1162,104 @@ def add_arguments(self, parser: ArgumentParser):
+                "messages to the queue using this generated key as the topic."
+            ),
+        )
+        parser.add_argument(
+            "-iq",
+            "--inbound-queue",
+            dest="inbound_queue",
+            type=str,
+            env_var="ACAPY_INBOUND_TRANSPORT_QUEUE",
+            help=(
+                "Specifies inbound queue connection details/host. The input for "
+                "this option takes the form `[protocol]://[host]:[port]`. So for "
+                "example, to use Redis on host `myredis.mydomain.com` the input "
+                "should be `redis://myredis.mydomain.com:port`. Currently, support "
+                "Redis and Kafka backend only."
+            ),
+        )
+        parser.add_argument(
+            "-iqc",
+            "--inbound-queue-class",
+            dest="inbound_queue_class",
+            type=str,
+            env_var="ACAPY_INBOUND_TRANSPORT_QUEUE_CLASS",
+            help=(
+                "Defines the location of the Inbound Queue Engine. This must be "
+                "a 'dotpath' to a Python module on the PYTHONPATH, followed by a "
+                "colon, followed by the name of a Python class that implements "
+                "BaseInboundQueue. This commandline option is the official entry "
+                "point of ACA-py's pluggable queue interface. The default value is: "
+                "'aries_cloudagent.transport.inbound.queue.redis.RedisInboundQueue'."
+            ),
+        )
+        parser.add_argument(
+            "-iqp",
+            "--inbound-queue-prefix",
+            dest="inbound_queue_prefix",
+            type=str,
+            env_var="ACAPY_INBOUND_TRANSPORT_QUEUE_PREFIX",
+            default="acapy",
+            help=(
+                "Specifies the queue topic prefix. The queue topic is generated "
+                "in the following form: 'prefix.inbound_transport'. The default value"
+                "is the value `acapy`, so a queue key of 'acapy.inbound_transport' "
+                "is generated in the case of the default settings. ACA-py will send "
+                "messages to the queue using this generated key as the topic."
+            ),
+        )
+        parser.add_argument(
+            "-iqt",
+            "--inbound-queue-transport",
+            dest="inbound_queue_transports",
+            type=str,
+            action="append",
+            nargs=3,
+            required=False,
+            metavar=("<module>", "<host>", "<port>"),
+            env_var="ACAPY_INBOUND_QUEUE_TRANSPORT",
+            help=(
+                "REQUIRED. Defines the inbound queue transport(s) on which the inbound "
+                "delivery_service agent listens for receiving messages from other "
+                "agents. This parameter can be specified multiple times to create "
+                "multiple interfaces. Built-in inbound transport types include "
+                "'http' and 'ws'."
```

Comment:
This help text isn't strictly accurate, is it? It is not required if `-it` is present?